src/templates/admin/copyediting/add_copyeditor_assignment.html

```diff
@@ -66,8 +66,8 @@ <h2>2. Select Files</h2>
                     </tr>
                     {% for file in article.manuscript_files.all %}
                         <tr>
-                            <td><input type="checkbox" name="files" value="{{ file.id }}"
-                                       {% if file.id in files %}checked="checked"{% endif %}></td>
+                            <td><input type="checkbox" name="files_for_copyediting" value="{{ file.id }}"
+                                       {% if file in form.cleaned_data.files_for_copyediting %}checked="checked"{% endif %}></td>
```

Comment:
Is it not possible to let django render the form here instead of having to re-implement its logic?

Reply:
It is possible, but it would only display the file name, not the full table we want. We use this a lot so I wonder if making a template that handles this would be useful.